src/main/java/utility/Parser.java

```diff
@@ -0,0 +1,32 @@
+package utility;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * A Parser class that is in charge of parsing user input.
+ */
+public class Parser {
+
+    /**
+     * Reads user input and takes the first word to convert it
+     * into ENUM format.
+     *
+     * @param input User input in string.
+     * @return Choice ENUM.
+     * @throws DukeException Duke Custom Exception.
+     */
+    public Choice parseCommand(String input) throws DukeException {
+        ArrayList<String> commands = new ArrayList<>(
+                Arrays.asList("DONE", "LIST", "DEADLINE", "EVENT", "TODO", "BYE", "DELETE", "FIND"));
```

Comment:
Perhaps these values could be obtained directly from the Choice Enum class like [here](https://stackoverflow.com/questions/13783295/getting-all-names-in-an-enum-as-a-string)! That would mean less updating 👍